src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -874,11 +874,8 @@ public string Remove(int startIndex, int count)
         // a remove that just takes a startindex.
         public string Remove(int startIndex)
         {
-            if (startIndex < 0)
-                throw new ArgumentOutOfRangeException(nameof(startIndex), SR.ArgumentOutOfRange_StartIndex);
-
-            if (startIndex >= Length)
-                throw new ArgumentOutOfRangeException(nameof(startIndex), SR.ArgumentOutOfRange_StartIndexLessThanLength);
+            if ((uint)startIndex > Length)
+                throw new ArgumentOutOfRangeException(nameof(startIndex), startIndex < 0 ? SR.ArgumentOutOfRange_StartIndex : SR.ArgumentOutOfRange_StartIndexLessThanLength);
```

Comment:
> ArgumentOutOfRange_StartIndexLessThanLength

Does this string need to be updated?

https://github.com/dotnet/runtime/blob/6791d05494f5aa4d1a789525c2708e775aafdc07/src/libraries/System.Private.CoreLib/src/Resources/Strings.resx#L1822-L1824

Reply:
I can stick up a quick PR to fix it, if you agree.

Reply:
Good catch. Yes, thanks.